src/main/java/org/jabref/gui/preferences/TablePrefsTab.java

```diff
@@ -54,25 +55,34 @@ public TablePrefsTab(JabRefPreferences prefs) {
         Label formatOfAuthor = new Label(Localization.lang("Format of author and editor names"));
         formatOfAuthor.getStyleClass().add("sectionHeader");
         builder.add(formatOfAuthor, 1, 1);
+        final ToggleGroup formatNamesToggleGroup = new ToggleGroup();
+        final ToggleGroup nameAbbrevToggleGroup = new ToggleGroup();
         builder.add(namesAsIs, 1, 2);
+        namesAsIs.setToggleGroup(formatNamesToggleGroup);
         builder.add(noAbbrNames, 2, 2);
+        noAbbrNames.setToggleGroup(nameAbbrevToggleGroup);
         builder.add(namesFf, 1, 3);
+        namesFf.setToggleGroup(formatNamesToggleGroup);
         builder.add(abbrNames, 2, 3);
+        abbrNames.setToggleGroup(nameAbbrevToggleGroup);
         builder.add(namesFl, 1, 4);
+        namesFl.setToggleGroup(formatNamesToggleGroup);
         builder.add(lastNamesOnly, 2, 4);
+        lastNamesOnly.setToggleGroup(nameAbbrevToggleGroup);
         builder.add(namesNatbib, 1, 5);
-
+        namesNatbib.setToggleGroup(formatNamesToggleGroup);
         Label label1 = new Label("");
         builder.add(label1, 1, 6);
-
         Label general = new Label(Localization.lang("General"));
         general.getStyleClass().add("sectionHeader");
         builder.add(general, 1, 7);
         builder.add(autoResizeMode, 1, 8);
-        namesNatbib.setOnAction(e -> {
-            abbrNames.setDisable(namesNatbib.isSelected());
-            lastNamesOnly.setDisable(namesNatbib.isSelected());
-            noAbbrNames.setDisable(namesNatbib.isSelected());
+
+        formatNamesToggleGroup.selectedToggleProperty().addListener((observable, oldValue, newValue) -> {
+            final boolean natBibIsSelected = newValue.equals(namesNatbib);
```

Comment:
you can use == as boolean is a primite

Reply:
Hmm, not sure i understand what you're saying. Do you propose to replace to the call to equals on newValue by == ? Wouldn't this compare simply the references and not the actual objects behind them? Could you please clarify this for me.

Reply:
Forget my comment. Did not see that the newValue is an Toggle object. I thought it's a boolean value. My fault.

Nonetheless you could directly bind the value of the control, you don't need the extra check then.
`abbrNames.disableProperty().bind(namesNatBIb.selectedProperty()) `

This will automatically change the state of the control, depending on the selectedProperty of the other control.
That's one of the nice javafx features ;)